python/tests/integration/test_client_server.py

```diff
@@ -9,11 +9,12 @@
 from tchannel.outgoing import OutgoingTChannel
 from tchannel.tornado import TChannel
 from tchannel.tornado.connection import TornadoConnection
+from tests.conftest import big_arg
```

Comment:
Once `big_arg` is a fixture, instead of importing it, just use it in functions that need it.